lib/proteus/kit.rb

```diff
@@ -16,7 +16,9 @@ def new(kit_name, repo_name = nil)
 
       if system "git ls-remote #{url(kit_name)} #{repo_name} > /dev/null 2>&1"
         puts "Starting a new proteus-#{kit_name} project in #{repo_name}"
-        system "git clone #{url(kit_name)}#{' ' + repo_name}"
+        system "git clone #{url(kit_name)}#{' ' + repo_name} && "\
+          "rm -rf #{repo_name}/.git && "\
+          "git init #{repo_name}"
```

Comment:
Should we also create an initial commit for them? Maybe with a message like "New Proteus-#{kit_name} project"

I'm not sure it would occur to me that I need to commit before changing things